all/projectAlcazar/src/app/project.c

```diff
@@ -35,6 +36,10 @@
 #define LED_RGB_GREEN_CH 1
 #define LED_RGB_BLUE_CH 0
 
+//HOMEWORK: PC8 + PC9 sunt definiti pe cananlul TIM3 (n-1 in functie de canal)
+#define LED_PIN_BLUE_CH 2
```

Comment:
From the datasheet section 4 table 13
```
PC8 I/O FT - TIM3_CH3 -
PC9 I/O FT - TIM3_CH4
```
So channels are actually 3 and 4

Reply:
shouldn't they be channel 2 & 3 ? based on the rgb LED the channels are 0,1,2 for timer1 channels 1,2 & 3. in Timer.c the switch case starts with 0 . 
switch(channel)
	{
		case 0:
			pDutyRegister = & (timer->CCR1);
			break;